package/com.unity.formats.usd/Documentation~/USD Import.md

```diff
@@ -0,0 +1,53 @@
+# Importing USD Files
+
+![](Images/USD_Menu.png)
+
+To import a USD file into Unity, use the USD context menu, and select one of the 'Import as ...' options as required. Supported USD file formats are:
+
+-   .usd
+-   .usda
+-   .usdc
+-   .usdz
+
+## Importing Options
+
+-   **Import As GameObjects**: Imports selected USD file as GameObjects directly into the Scene Hierarchy for the open scene.
+
+-   **Import as Prefab**: Imports selected USD file as a Prefab into the project folder. This method will also generate PlayableAssets and materials that can be reused. Note that the import will be much slower than for GameObjects, and the prefab must be unpacked to make any changes.
+
+-   **Import as Timeline Clip**: Imports selected USD file as a Prefab containing empty Materials and the Timeline Clip. If animation is recorded in the Timeline Clip, that can be used to animate other applicable Unity Objects.
+
+## Importing Materials
+
+To import materials from USD, firstly import the USD file using the USD menu. Then, to get the materials, change the 'Import Settings' > 'Materials' in the Inspector to 'Import Preview Surface'. Finally re-import the USD file.
```

Comment:
We can start this paragraph in the same way as "Importing Payloads" i.e. "Mateirals are not loaded by default".